docs/6-operating-guidelines.md

```diff
@@ -24,39 +24,41 @@ Antifragility and non-gameability emerge from simple frameworks. This limits the
   - `timeWeighting` can be either 1.0 for full-time or .5 for part-time contributors
   - Existing contributors get "diluted" as newcomers come in
   - Continuing contributors get additional weight per month they are active. This means historic contributors don't maintain their split weighting if they leave protocol development
-  - Anyone who stops contributing is removed from the split via periodic curation reviews
+  - Anyone who stops contributing should remove themselves from the membership, and may be removed via periodic curation reviews
 
-## 6.4 Proposing and Discussing New Members
+## 6.4 Modifying Projects and Members
 
-If current members believe that the set should be expanded to include another contributor, they should propose them well enough in advance of a membership update. This would mean at least a week. The process would look something like this:
+Modifying eligible projects and active membership should be proposed separately in the case both are being considered. Both should be shared with the membership at least two weeks in advance of any onchain update.
 
-1. An existing member (akin to an EIP champion) should send this info in the `proposed-members` channel.
+### Adding/removing Projects
+
+Changing the list of [eligible projects](link) can be made through a PR to the docs repo. This PR should add the project in the appropriate section, along with the following info:
+
+- Name of project/research area
+- Summary of why this project/research area should be considered eligible 
+
+### Adding Members
+
+Changing the current membership can be done by an existing member making a PR in the membership repo (currently private to members only). 
+
+When adding a new member, the PR should add the individual to the member list, along with the accompanying info:
 
 - Name / Identifier
-- Team / Project
-- Discord handle
+- Project
+- Discord handle 
 - Link to relevant work, eg. GitHub, research
 - Summary of their work and eligibility
 
-2. Discussion should be open for at least a few days to give members time to review and contribute to the discussion. This can be either with reacts on the proposal 👍 / 👎, ideally along with some written thoughts on why you think the proposed member fits the eligibility or not. Deliberations should strive to end in rough consensus, resorting to formal voting procedures only in rare extreme situations.
+Discussion should be open for at least one week to give members time to review and discuss. This can be with reacts on the proposal 👍 / 👎 or written thoughts on why the proposed member fits the eligibility or not.
 
-3. Once a positive consensus has been reached, the person can be contacted with the typical onboarding flow. 
+Once one week has passed, the PR should be merged.
```

Comment:
```suggestion
Once one week has passed, if no objections, the PR can be merged.
```

Reply:
does this need to cover the case where the proposed PR does not get merged?

Reply:
yep good catch!